molecule/default/tests/test_dashboard.py

```diff
@@ -4,22 +4,51 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
 @pytest.mark.jira('asc-222')
+# Experimental test
+def test_find_openrc_on_root(host):
```

Comment:
These are wrong. We do not know where the `openrc` file is located, therefore you cannot look just at `{/root,/opt,/etc}/openrc`. We need to find out where the `openrc file is located. Therefore we need to perform a find operation for these directories:
```
cmd = "find {/opt,/etc,/root} -name openrc -print | grep openrc"
host.run(cmd)
```

Reply:
Good point, but I rather split it into 3 different tests, because if it passed, we don't know where the openrc is located

Reply:
Ok, but your tests currently only test for `/root/openrc`, `/opt/openrc`, `/etc/openrc` and will fail if `openrc` is in a subdirectory below these locations (e.g. `/etc/openstack_deploy/openrc`). So, you would still need to perform a `find` operation for those locations to find out if the file is nested somewhere.

Reply:
yes, updated in the latest commit (4e40de1)